server/utils/env.ts

```diff
@@ -25,6 +25,12 @@ const WOWZA_SCP_PRIVATE_KEY =
 const WOWZA_SCP_PASS_PHRASE = process.env.WOWZA_SCP_PASS_PHRASE ?? "";
 const WOWZA_SCP_SERVER_PATH =
   process.env.WOWZA_SCP_SERVER_PATH ?? "/var/www/wowza-upload";
+const ZOOM_API_KEY = process.env.ZOOM_API_KEY ?? "";
+const ZOOM_API_SECRET = process.env.ZOOM_API_SECRET ?? "";
+const ZOOM_IMPORT_INTERVAL = process.env.ZOOM_IMPORT_INTERVAL ?? "";
+const ZOOM_IMPORT_TO = process.env.ZOOM_IMPORT_TO ?? "";
+const ZOOM_IMPORT_WOWZA_BASE_URL = process.env.ZOOM_IMPORT_WOWZA_BASE_URL ?? "";
+const ZOOM_IMPORT_AUTODELETE = process.env.ZOOM_IMPORT_AUTODELETE ?? "";
```

Comment:
本件対応しなくともよいですがstringではなくともたとえば下記のようにこの段階でbooleanに変換しても問題無さそうに見えました。

```suggestion
const ZOOM_IMPORT_AUTODELETE = Boolean(process.env.ZOOM_IMPORT_AUTODELETE);
```